youtube_dl/extractor/indavideo.py

```diff
@@ -1,6 +1,14 @@
 # coding: utf-8
 from __future__ import unicode_literals
 
+try:
+    # Python 3
+    from urllib.parse import urlsplit, urlunsplit, urlencode, parse_qsl
+except ImportError:
+    # Python 2
+    from urlparse import urlsplit, urlunsplit, parse_qsl
+    from urllib import urlencode
```

Comment:
`compat_*` functions.

Reply:
not necessary thanks to your helpful suggestion of `update_url_query` below